run/sense2john.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
Does this script support both Python 2 and 3? If so, great. If not, we need to specify `python2` or `python3` here.

Reply:
Passes basic testing with both Python 2 and 3 - great! - but I don't have a proper XML file to test this for real... which reminds me - please send us a sample file like this via a pull request against https://github.com/openwall/john-samples